src/transformers/__init__.py

```diff
@@ -131,6 +131,7 @@
     ],
     "models": [],
     # Models
+    "models.dbrx": ["DbrxConfig"],
```

Comment:
Perhaps you should add `DBRX_PRETRAINED_CONFIG_ARCHIVE_MAP` and make it
```python
"models.dbrx": ["DBRX_PRETRAINED_CONFIG_ARCHIVE_MAP", "DbrxConfig"]
```

This is what all the other models have

Reply:
Mentioned this below, but pretrained checkpoint maps like `DBRX_PRETRAINED_CONFIG_ARCHIVE_MAP` are no longer required, see [this PR](https://github.com/huggingface/transformers/pull/29112)!